vale/styles/vocab.txt

```diff
@@ -36,11 +37,13 @@ JWT
 JWTs
 Kuzco
 Memcache
+MacOS
 minitest
 misconfiguration
 Namespace
 Namespaced
 Namespaces
+Nginx
```

Comment:
This looks like it's stylised as either `nginx` or `NGINX`. It looks like the latter is more prominent these days.

Reply:
Let's chat through the rest of these during our call today.